README.md

```diff
@@ -1 +1,43 @@
 # AWeber FlexBox Grid
+AWeber FlexBox Grid is born from AWeber's need to have a responsive and modern grid system to use on its website and web application. This is why we used FlexBox grid as the base for this project but using SCSS as a preprocessor language to provide maintainability and ease of development.
+
+## Installation
+### npm
+TBD…
+
+### Bower
+TBD…
+
+### Production CSS | CDN
+TBD…
+
+## Development
+* Fork and clone the **AWeber FlexBox Grid** repository
+* Install project’s dependencies
+* Create a branch for your changes using [Git Flow](http://nvie.com/posts/a-successful-git-branching-model/)
+* Make your changes and document them properly using [SASSDoc](http://sassdoc.com/) guidelines
+* Lint your changes and create a PR for your changes pointing to `develop` branch
```

Comment:
`* Lint your changes before opening a PR.`
`* Open a PR comparing your branch with the `develop` branch as the base.`